tools/crm_mon_curses.c

```diff
@@ -425,7 +425,7 @@ static int
 cluster_maint_mode_console(pcmk__output_t *out, va_list args) {
     unsigned long long flags = va_arg(args, unsigned long long);
 
```

Comment:
guess this is the first 2023 edit in crm_mon_curses.c

Reply:
thanks